### DISCUSSION TITLE
A voter survey

### DISCUSSION THREAD

**Discussion by aleeusgr** [cardano-foundation/state-of-the-developer-ecosystem#1] [F.A.Q.]
What do you think about a Voter Survey? I believe ecosystem will benefit if there would be an open document where voter preferences are available to review. The questions to ask are an open question, something about how voters make decisions. The goal is to help proposers build an image of a voter, the audience to which a Catalyst proposal is supposed to speak to.

Many thanks for the State of the Developer Ecosystem Survey, I am reviewing it and I propose Developer Experience working group should do this yearly and use it to guide the group decision making.
 https://github.com/input-output-hk/Developer-Experience-working-group/discussions/69

**Comment by KtorZ** [comment]
Hey, first, sorry for the delay in answering. I somehow missed any notification coming from Github here.

 I am not sure to fully understand the idea of a "Voter Survey"? By "Voter", do you mean specifically Catalyst voter or are you targeting a broader audience such as any future CIP-1694 voter/drep? What would be the end goal?

**Reply by aleeusgr** [reply]
np.
Thinking back with the knowledge I acquired meanwhile I think this is technically impossible.
I was thinking is to rank wallets by voting power, come with some reasonable number and reach out directly.
If only life was as simple
